src/components/systemButtons/systemButtons.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { cva, VariantProps } from 'class-variance-authority';
+
+const buttonVariants = cva(
+  [
+    'rounded-lg flex items-center justify-center cursor-pointer',
+    'transition-all duration-300 ease-in-out focus:outline-none',
+  ],
+  {
+    variants: {
+      size: {
+        large: 'w-[170px] h-[56px]',
+        medium: 'w-[170px] h-[42px]',
+        // small: '',
+      },
+      variant:{
+        filled: {
+            large: 'bg-primary-400',
+            medium: 'bg-primary-400',
+            // small: '',
+        },
+        outline: {
+            large: 'border border-primary-400',
+            medium: 'border border-primary-400',
+            // small: '',
+        },
+        ghost: {
+            large: 'bg-transparent',
+            medium: 'bg-transparent',
+            // small: ''
+        },
+      },
+      state: {
+        default: 'bg-primary-400',
+        ghostdefault: 'bg-transparent',
+        ghosthover: 'bg-gray-400',
```

Comment:
Hover is not a state. The button should change color when is being hovered. You can perform this with `hover:` in tailwind. For example
<button className="bg-primary-400 hover:bg-primary-500`. The button will be primary-400, but if you hover it, it will change to primary-500